[2021-02-06 13:21:17,585] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-04T00:00:00+00:00 [queued]>
[2021-02-06 13:21:17,595] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-04T00:00:00+00:00 [queued]>
[2021-02-06 13:21:17,596] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 13:21:17,596] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-06 13:21:17,597] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-06 13:21:17,613] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_0> on 2021-02-04T00:00:00+00:00
[2021-02-06 13:21:17,618] {standard_task_runner.py:52} INFO - Started process 63 to run task
[2021-02-06 13:21:17,622] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-02-04T00:00:00+00:00', '--job-id', '4', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpytv_xtyr', '--error-file', '/tmp/tmp__h9ciq1']
[2021-02-06 13:21:17,622] {standard_task_runner.py:77} INFO - Job 4: Subtask runme_0
[2021-02-06 13:21:17,673] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_0 2021-02-04T00:00:00+00:00 [running]> on host 66bc8b98f452
[2021-02-06 13:21:17,792] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-02-04T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-04T00:00:00+00:00
[2021-02-06 13:21:17,793] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-06 13:21:17,794] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_0__20210204" && sleep 1
[2021-02-06 13:21:17,803] {bash.py:169} INFO - Output:
[2021-02-06 13:21:17,804] {bash.py:173} INFO - example_bash_operator__runme_0__20210204
[2021-02-06 13:21:18,805] {bash.py:177} INFO - Command exited with return code 0
[2021-02-06 13:21:18,829] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20210204T000000, start_date=20210206T132117, end_date=20210206T132118
[2021-02-06 13:21:18,856] {taskinstance.py:1219} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-06 13:21:18,877] {local_task_job.py:146} INFO - Task exited with return code 0
